js/application.js

```diff
@@ -1 +1,4 @@
 // instantiate a StockView
+
+var stock = new Stock(symbol)
+var stockView = new StockView(stock)
```

Comment:
i think we potentially don't even need to create a new stock here. I think we should be able to create a new stock in our view layer, we could add an event listener to the StockView that allows us to create a new symbol on the lookup.